shared/common-adapters/button.js

```diff
@@ -124,69 +107,79 @@ const fullWidthHeight = isMobile ? 48 : 32
 const borderRadius = 50
 const smallBorderRadius = isMobile ? 50 : 28
 
-const common = {
-  ...globalStyles.flexBoxColumn,
-  alignItems: 'center',
-  alignSelf: 'center',
-  borderRadius,
-  height: regularHeight,
-  justifyContent: 'center',
-  paddingLeft: globalMargins.medium,
-  paddingRight: globalMargins.medium,
-  ...(isMobile
-    ? {}
-    : {
-        display: 'inline-block',
-        lineHeight: 'inherit',
-      }),
-}
-const commonLabel = {
-  color: globalColors.white,
-  textAlign: 'center',
-  ...(isMobile
-    ? {}
-    : {
-        whiteSpace: 'pre',
-      }),
-}
-const fullWidth = {
-  alignSelf: undefined,
-  height: fullWidthHeight,
-  width: null,
-}
+const styles = styleSheetCreate({
+  common: platformStyles({
+    common: {
+      ...globalStyles.flexBoxColumn,
```

Comment:
It's okay to spread styles within `styleSheetCreate`?

Reply:
Yup, `globalStyles.*` are plain objects. Spreading things _created_ with `styleSheetCreate` is no good. Spreading these is bad practice though so I'll switch it to `collapseStyles`. Thanks!